hadoop/src/main/java/org/apache/carbondata/hadoop/api/AbstractDataMapJob.java

```diff
@@ -36,7 +36,7 @@
   }
 
   @Override public List<ExtendedBlocklet> execute(DistributableDataMapFormat dataMapFormat,
-      FilterResolverIntf resolverIntf) {
+      FilterResolverIntf resolverIntf, CarbonTable carbonTable) {
```

Comment:
No need to pass carbonTable here, it is already present in `DistributableDataMapFormat`, you can get from there